Modules/mathmodule.c

```diff
@@ -2016,6 +2016,63 @@ math_factorial(PyObject *module, PyObject *arg)
 }
 
 
+/*[clinic input]
+math.lcm
+
+    x as a: object
+    y as b: object
+    /
+least common multiple of x and y
+[clinic start generated code]*/
+
+static PyObject *
+math_lcm_impl(PyObject *module, PyObject *a, PyObject *b)
+/*[clinic end generated code: output=6f83fb6d671074ba input=bd41b785dc2a4ff1]*/
+
+static PyObject*
+math_lcm(PyObject *module, PyObject *a, PyObject *b)
```

Comment:
This line and the line immediately above it should be removed. This is the current cause of the build failing.